### ISSUE TITLE
Copyright violation

### ISSUE THREAD

**Issue by Poussinou** [derekcsm/hubble_gallery#53]
Hi @derekcsm 

I found [this app ](https://play.google.com/store/apps/details?id=com.ikramshah.hubble_gallery) and also [this one](https://play.google.com/store/apps/details?id=com.darklydreaming.spaceimages) on the Google Play Store. This guy just took the source code and put it on the Store, without warning this is free software, and one of them added ads...

As you are the owner of the source code, feel free to open a request to google [here](https://support.google.com/legal/contact/lr_dmca?product=googleplay) for each one, to remove them from the Store, if it's your wish ;)

Don't forget to tell Google that:

* The app is licensed under Apache license and this guy is violating it;
* He is falsely claiming he made the app...

**Comment by IzzySoft** [comment]
While both apps have meanwhile been unpublished from Playstore – are you still maintaining this app, @derekcsm? The last commit was more than 2 years in the past, so I'm just wondering …

**Comment by derekcsm** [comment]
@IzzySoft I'm not unfortunately because of other interests, but if you or anyone willing/able would like to take this project on I would be open to transferring ownership to someone who can keep it going.

**Comment by IzzySoft** [comment]
Thanks for your reply, @derekcsm! I'm no dev, so I cannot take over (my reason to ask [is here](https://apt.izzysoft.de/fdroid/index/apk/com.derek_s.hubble_gallery)). But could you do one last release, removing Crashlytics? Then I could at least remove the `Tracking` AntiFeature.

**Comment by Poussinou** [comment]
@IzzySoft this app doesn't work anymore, you should remove it from your repo, and we should archive it in the official main repo

**Comment by IzzySoft** [comment]
@Poussinou thanks for letting me know! That confirmed, or just doesn't work for you? If confirmed (generally doesn't work anymore, e.g. because of API/URL changes having happened after the last release) I'll remove it from my repo ASAP and would open an MR to archive it on main.

**Comment by Poussinou** [comment]
I tried on two different phones both the version of your repo and the one from F-Droid main, and still both don't work, you can go ahead 👍

**Comment by IzzySoft** [comment]
Thanks @Poussinou – confirmed: installs fine, starts, intro OK … no crashes, but no content either. Logcat gives no clue but I assume the images have been (re)moved. So I removed the app for now as well (to avoid disappointed users).

@derekcsm should someone turn up, taking over and providing a working version again, I gladly re-establish the app in my repo. Metadata have been backed up, so it should be quite easy then.

**Comment by derekcsm** [comment]
Hey guys, sorry for the late update I've been offline in Ireland for a while! Looks like the website that the app scrapes HTML from has changed and it would require some work to get a version properly supported and working again. 

Just in an airport right now, but I started working on this branch a year ago to modernize before I ended up being too busy with other things to finish it. It might be something to try to finish if someone wants to test out the build on that branch and finish it. https://github.com/derekcsm/hubble_gallery/tree/modernize
